planner/core/rule_eliminate_projection.go

```diff
@@ -63,15 +63,16 @@ func resolveColumnAndReplace(origin *expression.Column, replace map[string]*expr
 	}
 }
 
-func resolveExprAndReplace(origin expression.Expression, replace map[string]*expression.Column) {
+// ResolveExprAndReplace replaces columns fields of expressions by children logical plans
```

Comment:
nit: Comment should always be a complete sentence, which should end with a '.'.
So you'd better add a period at the end.

Reply:
Comment addressed. PTAL, thanks~ @francis0407